<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEmployeeDialog</class>
 <widget class="QDialog" name="EditEmployeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактировать сотрудника</string>
  </property>
  <widget class="QLabel" name="labelHeader">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>10</y>
     <width>601</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактирование данных сотрудника</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>581</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelSurname">
      <property name="text">
       <string>Фамилия:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditSurname"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>Имя:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditName"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPatronymic">
      <property name="text">
       <string>Отчество:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditPatronymic"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelPassport">
      <property name="text">
       <string>Паспорт:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditPassport"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelJobTitle">
      <property name="text">
       <string>Должность:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelMedicalPolicy">
      <property name="text">
       <string>Медицинский полис:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditMedicalPolicy"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelInsurancePolicy">
      <property name="text">
       <string>Страховой полис:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditInsurancePolicy"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelVacationStart">
      <property name="text">
       <string>Дата начала отпуска:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDateEdit" name="dateEditVacationStart">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="labelVacationEnd">
      <property name="text">
       <string>Дата окончания отпуска:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDateEdit" name="dateEditVacationEnd">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditJobTitle"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
